todo_app/data/session_items.py

```diff
@@ -70,17 +70,13 @@ def save_item(item):
 
 def delete_item(id):
```

Comment:
You can get rid of session_items.py now, we will not need it again.

Reply:
Removed